tracer/src/Datadog.Trace/Ci/CIVisibility.cs

```diff
@@ -25,19 +25,24 @@ namespace Datadog.Trace.Ci
 {
     internal class CIVisibility
     {
-        private static readonly Lazy<bool> _enabledLazy = new(InternalEnabled, true);
-        private static CIVisibilitySettings _settings = CIVisibilitySettings.FromDefaultSources();
+        private static readonly Lazy<bool> EnabledLazy = new(InternalEnabled, true);
```

Comment:
We usually add the _ prefix to private fields. In the case of private static readonly fields, watching the code, we have several examples of both cases. Do we follow a particular naming style regarding prefixes for those?